### PR TITLE
scantailor: Add original and two forks

### LINKED ISSUE #2368: [request] Add ScanTailor.
Scan Tailor is an interactive post-processing tool for scanned pages. It performs operations such as page splitting, deskewing, adding/removing borders, and others. You give it raw scans, and you get pages ready to be printed or assembled into a PDF or DJVU file. Scanning, optical character recognition, and assembling multi-page documents are out of scope of this project.
site:http://scantailor.org/

### PR DESCRIPTION
**Pull request by hoilc**
Fix #2368

### CODE REVIEW

File: bucket/scantailor-advanced.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            "hash": "c73a46f3a2f3d99b434a8724c33d7b9745fa90253cdae8640dd99fac9793b032"
+        }
+    },
+    "shortcuts": [
+        [
+            "scantailor.exe",
+            "ScanTailor Advanced"
+        ]
+    ],
+    "pre_install": "if(!(Test-Path(\"$persist_dir\\config\"))) { New-Item -ItemType \"Directory\" \"$dir\\config\" | Out-Null }",
```
Comment (Ash258) [inline bucket/scantailor-advanced.json]: Folders do not have to be precreated.

```suggestion
```

Comment (Ash258) [reply bucket/scantailor-advanced.json]: Sidenote for future. Do not use `"` for simple strings. Use `'` or do not use at all if it is enum. Like here.

Powershell is designed to work without quotes. Quotes are needed only if it is not simple string (one word string)

Comment (hoilc) [reply bucket/scantailor-advanced.json]: got it. 😄

**Comment by hoilc** [comment]
> This is use case for versions bucket I think.

 It's a little difficult to choose one survivor between scantailor-advanced and scantailor-universal.

Scantailor Advanced is more actively developing last year while Scantailor Universal just releases new version after long periods of inactivity.

And I made the choice. 😄

File: bucket/scantailor-advanced.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        "64bit": {
+            "url": "https://github.com/4lex4/scantailor-advanced/releases/download/v1.0.16/scantailor-advanced-1.0.16-win64.exe#!/dl.7z",
+            "hash": "ce3c1bae867ccaf020c759efad463ae0b937fac5ea3ef8fc4ce559ddd1ae8fc9"
+        },
+        "32bit": {
+            "url": "https://github.com/4lex4/scantailor-advanced/releases/download/v1.0.16/scantailor-advanced-1.0.16-win32.exe#!/dl.7z",
+            "hash": "c73a46f3a2f3d99b434a8724c33d7b9745fa90253cdae8640dd99fac9793b032"
+        }
+    },
+    "shortcuts": [
```
Comment (Ash258) [inline bucket/scantailor-advanced.json]: - No bin
- Remove unneeded files

```suggestion
    "post_install": "Remove-Item \"$dir\\`$*\" -Recurse",
    "bin": "scantailor.exe",
    "shortcuts": [
```

https://github.com/scantailor/scantailor/blob/master/CommandLine.cpp
https://github.com/4lex4/scantailor-advanced/blob/master/CommandLine.cpp

File: bucket/scantailor-advanced.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{
+    "description": "A fork of ScanTailor that merges the features of the ScanTailor Featured and ScanTailor Enhanced versions, brings new ones and fixes.",
```
Comment (Ash258) [inline bucket/scantailor-advanced.json]: This description do not state what scintailor is in general.

Something like `Interactive post-processing tool for scanned pages`

File: bucket/scantailor-advanced.json (hunk 2 above)
Comment (hoilc) [reply bucket/scantailor-advanced.json]: 😂Both original scantailor and another fork have a `scantailor-cli` executable as `bin`, but scantailor-advanced doesn't.

**Pull request merged**